src/core/client/auth/local/initLocalState.ts

```diff
@@ -1,28 +1,33 @@
 import { commitLocalUpdate, Environment } from "relay-runtime";
 
 import { parseQuery } from "talk-common/utils";
-import {
-  createAndRetain,
-  LOCAL_ID,
-  LOCAL_TYPE,
-} from "talk-framework/lib/relay";
+import { TalkContext } from "talk-framework/lib/bootstrap";
+import { initLocalBaseState, LOCAL_ID } from "talk-framework/lib/relay";
 
 /**
  * Initializes the local state, before we start the App.
  */
-export default async function initLocalState(environment: Environment) {
-  commitLocalUpdate(environment, s => {
-    const root = s.getRoot();
+export default async function initLocalState(
+  environment: Environment,
+  context: TalkContext
+) {
+  const authToken = window.location.hash
```

Comment:
Why not extract this to a separate function?

Reply:
Sure